src/components/Person/Person.jsx

```diff
@@ -1 +1,36 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+  const partner
+    = (sex === 'm')
+      ? 'wife'
+      : 'husband';
```

Comment:
Don't start the new line with =
```suggestion
  const partner = (sex === 'm')
      ? 'wife'
      : 'husband';
```